$MirOS: ports/print/cups/patches/patch-Makedefs_in,v 1.5 2006/11/08 18:25:43 bsiegert Exp $
--- Makedefs.in.orig	Mon Mar 12 21:48:50 2007
+++ Makedefs.in	Thu Mar 15 19:11:58 2007
@@ -28,8 +28,8 @@
 
 AR		=	@AR@
 AWK		=	@AWK@
-CC		=	@LIBTOOL@ @CC@
-CXX		=	@LIBTOOL@ @CXX@
+CC		=	@CC@
+CXX		=	@CXX@
 DSO		=	@DSO@
 HTMLDOC		=	@HTMLDOC@
 INSTALL		=	@INSTALL@
@@ -49,13 +49,13 @@ STRIP		=	@STRIP@
 # Installation programs...
 #
 
-INSTALL_BIN	=	$(LIBTOOL) $(INSTALL) -m 755
+INSTALL_BIN	=	$(LIBTOOL) --mode=install $(INSTALL) -m $(BINMODE)
 INSTALL_CONFIG	=	$(INSTALL) -m @CUPS_CONFIG_FILE_PERM@
-INSTALL_DATA	=	$(INSTALL) -m 644
+INSTALL_DATA	=	$(INSTALL) -m $(SHAREMODE)
 INSTALL_DIR	=	$(INSTALL) -d
-INSTALL_LIB	=	$(LIBTOOL) $(INSTALL) -m 755
-INSTALL_MAN	=	$(INSTALL) -m 644
-INSTALL_SCRIPT	=	$(INSTALL) -m 755
+INSTALL_LIB	=	$(LIBTOOL) --mode=install $(INSTALL) -m $(BINMODE)
+INSTALL_MAN	=	$(INSTALL) -m $(SHAREMODE)
+INSTALL_SCRIPT	=	$(INSTALL) -m $(BINMODE)
 
 #
 # Default user, group, and system groups for the scheduler...
@@ -197,6 +197,7 @@ sbindir		=	@sbindir@
 sharedstatedir	=	@sharedstatedir@
 srcdir		=	@srcdir@
 sysconfdir	=	@sysconfdir@
+datarootdir	=	@datarootdir@
 top_srcdir	=	@top_srcdir@
 
 BUILDROOT	=	$(DSTROOT)
@@ -241,12 +242,11 @@ DBUSDIR		=	@DBUSDIR@
 # Rules...
 #
 
-.SILENT:
 .SUFFIXES:	.1 .1.gz .1m .1m.gz .5 .5.gz .7 .7.gz .8 .8.gz .a .c .cxx .h .man .o .32.o .64.o .gz
 
 .c.o:
-	echo Compiling $<...
-	$(CC) $(ARCHFLAGS) $(OPTIM) $(CFLAGS) -c $<
+	@echo Compiling $<...
+	$(LIBTOOL) --mode=compile $(CC) $(ARCHFLAGS) $(OPTIM) $(CFLAGS) -c $<
 
 .c.32.o:
 	echo Compiling 32-bit $<...
@@ -257,8 +257,8 @@ DBUSDIR		=	@DBUSDIR@
 	$(CC) $(ARCH64FLAGS) $(OPTIM) $(CFLAGS) -c -o $@ $<
 
 .cxx.o:
-	echo Compiling $<...
-	$(CXX) $(ARCHFLAGS) $(OPTIM) $(CXXFLAGS) -c $<
+	@echo Compiling $<...
+	$(LIBTOOL) --mode=compile $(CXX) $(ARCHFLAGS) $(OPTIM) $(CXXFLAGS) -c $<
 
 .man.1 .man.1m .man.5 .man.7 .man.8:
 	echo Linking $<...
